aeolus/aeolus.cpp

```diff
@@ -257,10 +271,11 @@ SynthesizerGroup Aeolus::state() const
 //   setState
 //---------------------------------------------------------
 
-void Aeolus::setState(const SynthesizerGroup& g)
+bool Aeolus::setState(const SynthesizerGroup& g)
       {
       for (const IdValue& v : g)
             setValue(v.id, v.data.toDouble());
+      return true;
```

Comment:
Why make this change? Is there a way to return `false`?

Reply:
I second this question ;-), setters usually don't retrurn any value

Reply:
This is necessary because of changes to the Synthesizer interface.  See line 72 of synthesizer/synthesizer.h.